python_modules/dagster/dagster/core/storage/runs/schema.py

```diff
@@ -86,6 +86,8 @@
     db.Column("status", db.String(255), nullable=False),
     db.Column("timestamp", db.types.TIMESTAMP, nullable=False),
     db.Column("body", db.Text),
+    db.Column("action_type", db.String(255)),
```

Comment:
string/text and mysql index length seem reasonable?

Reply:
It seems fine.  Honestly, could probably do 32 or 64, considering that it's probably just a string that says "BACKFILL" or something like that.

Reply:
Also, it's a bit weird that the string is 255 here and the mysql_length is 32?